codegen/builtin_fs.go

```diff
@@ -191,7 +192,36 @@ func (l Local) Call(ctx context.Context, cln *client.Client, val Value, opts Opt
 
 		// When localPath is a filename instead of a directory, include and exclude
 		// patterns should be ignored.
-		localOpts = append(localOpts, llb.IncludePatterns([]string{filename}), llb.ExcludePatterns([]string{}))
+		localOpts = []llb.LocalOption{
```

Comment:
Why is it okay to overwrite `localOpts` here?

Reply:
`llb.IncludePatterns` and `llb.ExcludePatterns` overrides under the hood, not appends.

The change is equivalent, just more explicit.

Reply:
It seems weird that we'd ignore the options provided in the `opts` arg to this function and `SourceMap` when `localPath` is not a dir. Does that just happen to be the correct behavior for all the options we currently support? It seems very subtle.

Reply:
You're right, it's missing SourceMap. I'll change this back to append.